### PR TITLE
Mcs example helm

### PR DESCRIPTION
**Pull request by jaxels10**
This PR restructures the cluster env repo to contain 2 charts. One for Nidhogg and one for Yggdrasil. You can read about these charts in the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
+<img src="docs/images/Energinet-logo.png" width="250" style="margin-bottom: 3%">
+
 # cluster-env-repo
-This is the repository for the cluster environment
+This is the repository for the cluster environment. It contains two Helm charts called Nidhogg and Yggdrasil.
+
```
Comment (MartinSchmidt) [inline README.md:5]: Missing Principles -> DevOps
define appRep envRep and clusterRep

File: README.md [hunk 2]
```diff
@@ -1,2 +1,60 @@
...
+## Workflow
```
Comment (MartinSchmidt) [inline README.md:6]: Move workflow ("How to add an application") to the bottom.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,60 @@
...
+
+### Step one
+The developers of either a 3rd party application or the maintainers of the cluster should create a new application reposity that contains the code for their app. When this code is committed, it should trigger a build pipeline that will update the artifact repository. After this, the environment repository will need to be either manually or automatically updates to reflect the new artifacts. 
+
+### Step two
+Now that the application is created, the developers needs to create a pull request to the cluster env repository. This pull request should add the code block mentioned in [Yggdrasil](#yggdrasil). This PR will then need to be approved by the cluster development team, before it is merged into cluster env repository. When this is merged, ArgoCD will automatically deploy the application onto the cluster. 
+When the deployment is done, ArgoCD will poll the environment repository every 3 minutes, to check for changes to the application. 
+
+## Cluster
+When Nidhogg is deployed onto the cluster, it will create both the argoCD deployment as well as the Vnet needed for Kubernetes. This can seen in the following image.
```
Comment (MartinSchmidt) [inline README.md:18]: Vnet -> CNI

File: nidhogg/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+description: A chart that deploys argo
+version: 0.1
+
+dependencies:
+  - name: argo-cd
+    version: "2.17.1"
+    repository: "https://argoproj.github.io/argo-helm"
+  - name: mukube-vnet
+    version: "v0.1.2"
+    repository: "https://distributed-technologies.github.io/helm-repository/"
```
Comment (MartinSchmidt) [inline nidhogg/Chart.yaml:13]: add `condition: 'installCNI'`

File: nidhogg/values.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# A reference to this repo itself.
+yggdrasil:
+  repoURL: "https://github.com/distributed-technologies/cluster-env-repo.git"
+  targetRevision: "mcs-example-helm"
+
```
Comment (MartinSchmidt) [inline nidhogg/values.yaml:5]:
```suggestion
installCNI: true
```

**Review by MartinSchmidt — COMMENTED** [review]
I left a few comments to make the description sharper

File: README.md (hunk 3 above)
Comment (jaxels10) [reply README.md:18]: Corrected.

File: nidhogg/Chart.yaml (hunk 4 above)
Comment (jaxels10) [reply nidhogg/Chart.yaml:13]: Added

File: nidhogg/values.yaml (hunk 5 above)
Comment (jaxels10) [reply nidhogg/values.yaml:5]: Added

File: README.md (hunk 1 above)
Comment (jaxels10) [reply README.md:5]: Done

File: README.md (hunk 2 above)
Comment (jaxels10) [reply README.md]: Done

**Review by lynderup — APPROVED** [review]
Looks fine

**Pull request merged**